integration_tests/src/test/scala/com/nvidia/spark/rapids/JoinsSuite.scala

```diff
@@ -91,4 +95,42 @@ class JoinsSuite extends SparkQueryCompareTestSuite {
     mixedDfWithNulls, mixedDfWithNulls, sortBeforeRepart = true) {
     (A, B) => A.join(B, A("longs") === B("longs"), "LeftAnti")
   }
+
+  test("broadcast hint isn't propagated after a join") {
+    import spark.sqlContext.implicits._
+    spark.conf.set("spark.sql.autoBroadcastJoinThreshold", "-1")
```

Comment:
Won't setting configs on the shared spark session be problematic with the potential of config settings from one test leaking into other tests?  That's why Spark's tests use something like `withSQLConf` to only temporary change configurations for a specific test.